types/nwmatcher/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for nwmatcher 1.4
+// Project: https://github.com/dperini/nwmatcher
+// Definitions by: Wouter van Heeswijk <https://github.com/woutervh->
```

Comment:
As per https://github.com/dperini/nwmatcher#installation this can be used globally and is available as `NW.Dom`. That needs to be reflected in the definition. Please refer https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module for more information.